source/MRMesh/MRObjectLoad.cpp

```diff
@@ -342,6 +342,8 @@ static std::string makeWarningString( int skippedFaceCount, int duplicatedVertex
             res += '\n';
         res += fmt::format( "{} vertices were duplicated to make them manifold.", duplicatedVertexCount );
     }
+    if( holesCount )
+        res = fmt::format( "the objects contains {} holes. Please use the Mesh Healer plugin for fix this.", holesCount );
```

Comment:
`Please consider using Fill Holes tool` (dont use `plugin` and better to say `consider using` instead of `use`)